docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -5478,7 +5476,7 @@ ISNULL(last_name, 'Smith')
 
 The JULIANTIMESTAMP function converts a datetime value into a 64-bit
 Julian timestamp value that represents the number of microseconds that
-have elapsed between 4713 B.C., January 1, 00:00, and the specified
+have elapsed between 4713 B.C., January 1, 00:00:00, and the specified
```

Comment:
It is more common in written English to put the year after the date, thus: "January 1, 4713 B.C., 00:00:00".

Reply:
Thanks Dave, I've corrected.